lib/redis.js

```diff
@@ -74,19 +74,32 @@ class RedisClient {
 				try {
 					let command = line.trim();
 					let commands = splitargs(command);
-					let CMD = commands.shift().toLowerCase();
-					let func = this._redis_client[`${CMD}Async`];
-					if (typeof func == "function") {
-						let result = await func.bind(this._redis_client)(...commands);
-						if (Array.isArray(result)) {
-							result.each((item, index) => {
-								console.log("%d) %s", index, item);
+					if (commands.length !== 0) { // we have commands, so process, otherwise just a new prompt
+						let CMD = commands.shift().toLowerCase();
+						//`exit` and `clear` are not true commands, just part of REPL
+						if (CMD === 'exit') {
+							this._redis_client.quit(() => {
+								process.exit(0);
 							});
+						} else if (CMD === 'clear') {
+							console.log('\n'.repeat(process.stdout.rows))
```

Comment:
@stockholmux I think we should move to use an alternative clear method, like https://www.npmjs.com/package/clear

Reply:
changed to ANSI clear screen.